test/test_item.rb

```diff
@@ -170,6 +170,15 @@ def test_exchange_token_invalid_public_token
     end
   end
 
+  def test_public_token_sandbox
+    public_token_response = client.sandbox.sandbox_public_token.create(
+      SANDBOX_INSTITUTION,
```

Comment:
Ruby is different from Python, you'd need `create(institution_id: SANDBOX_INSTITUTION, initial_products: ['transactions']` here. Or make these two params regular params, without the colon.

Reply:
thanks for the review! i've updated the code with the colons, but can't test it since i don't have direct integrations enabled.